jolpica/formula_one/models/round.py

```diff
@@ -4,6 +4,8 @@
 
 from django.db import models
 
+from jolpica.formula_one.models.session import SessionType
```

Comment:
Please run `make format`, this will clean up and lint the code base.
Once I get around to it, it'll also be a requirement to merge via github actions checks.